src/capitalize.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * A function equivalent to TypeScript’s built-in `Capitalize` utility type.
```

Comment:
We don't need to mention "a function" because everything in Remeda is a function. Assuming people don't know Typescript's Capitalize util, it would be easier for people to get the full description for this function right here...

Suggestion (via chatgpt):
> Converts the first character of a given string to uppercase while maintaining the rest of the string as is.